src/routes/profile.svelte

```diff
@@ -0,0 +1,138 @@
+<script>
+import Input from '../components/atoms/Input.svelte'
+import Image from '../components/atoms/Image.svelte'
+import Button from '../components/atoms/Button.svelte'
+import Alert from '../components/atoms/Alert.svelte'
+import ActionButton from '../components/atoms/ActionButton.svelte'
+import { EditIcon, RotateCcwIcon } from 'svelte-feather-icons'
+
+let alertOpen = false
+let alertType = 'success'
+let alertMessage = ''
+let isEditing = false
+
+let form = {
+  name: '',
+  lastName: '',
+  email: '',
+  password: '',
+  passwordConfirm: '',
+}
+
+
+function handleSubmit () {
+  alertMessage = 'Usuario actualizado.'
+  if (form.password !== form.passwordConfirm) {
+    alertType = 'error'
+    alertMessage = 'La contraseña no coincide.'
+  }
+  alertOpen = true
+}
+
+function handleAlertClose () {
+  alertOpen = false
+  alertType = 'sucess'
+}
+
+function handleInputChange (event) {
+  const { name, value } = event.target
+  form[name] = value
+}
+
+function toggleIsEditing () {
+  isEditing = !isEditing
+}
+
+</script>
+
+<style>
+  .profile_-container {
```

Comment:
I think here is a typo

Reply:
You're right, I already uploaded a fix for it.